circle.yml

```diff
@@ -1,4 +1,32 @@
+machine:
+  timezone: UTC
+  java:
+    version: oraclejdk8
+
+# Collect build artifacts
+general:
+  artifacts:
+    - target/riffraff
 
 dependencies:
   pre:
     - ./setup.sh
```

Comment:
@clloyd do you think we need `./setup.sh` on build anymore? Or can we replace this with just a `npm install` as we have a post-install hook to do jspm things?